docs/AboutUs.adoc

```diff
@@ -4,53 +4,40 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-The dummy content given below serves as a placeholder to be used by future forks of the project. +
+BoNUS is currently being developed by the <<Project Team>> shown below based on the previous work of https://se-edu.github.io/docs/Team.html[se-edu] team. +
 {empty} +
-We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+We are a team based in the http://www.comp.nus.edu.sg[School of Computing], http://www.nus.edu.sg[National University of Singapore].
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Niu Yunpeng
+image::yunpengn.jpg[width="150", align="left"]
+{empty}[https://yunpengn.github.io/[homepage]] [https://github.com/yunpengn[github]]
 
-Role: Project Advisor
+Role: Developer
```

Comment:
Consider adding another line detailing your responsibilities. E.g. `Responsibilities: UI Component, Project management`

Reply:
Hi, Si Kai!

Thanks for the review. Sure, will add details to responsibilities.

Regarding the unicode issue:

Are you using Mac? I tested on a Mac laptop just now and found the same issue. I felt it is likely Mac OS misses that font.

If you are also using Mac, it is likely a compatibility issue and I will switch to another font.